RockPaperScissors.py

```diff
@@ -1,44 +1,117 @@
 ##
 #
 # Author: FX Coding Club
+#
+# Edited by Luke Sequeira
+#
 # Date 9/24/2020
 #
 
+#
+# 1 => rock
+# 2 => paper
+# 3 => scissors
+#
+#
+
+import random
+from libs import playsound
+
+sound = True
+
+score = 0 # Scores and
+games = 0 # Game played
+
+foo = input("Play with sound? y or n (if you do not choose a valid option, y is selected by default)")
+if(foo == 'n'):
+    sound = False
+    print("Sound has been turned off")
 
-# These are the possible choices in the game, using the randint function, the computer will be a choice based on the integer it chooses
-list = ['rock', 'paper', 'scissors']
+print("Welcome to Rock-Paper-Scissors. Type end anytime to exit.")
+def game():
 
+    global score
+    global games
+    global sound
+    
+    games += 1
+    
+    random.seed()
+    computerChoice = random.randint(1, 3)
 
-# Compare var1 and computerChoice to see who wins
+    #get a text value from the int chosen
+    if(computerChoice == 1):
+        comptext = "rock"
+    if(computerChoice == 2):
+        comptext = "paper"
+    if(computerChoice == 3):
+        comptext = "scissors"
 
-# The following is all the comparisons if the computer chooses "Rock"
-if computerChoice == 'rock' and var1 == 'scissors'):
-    print('You chose ' + var1 + ' and the computer chose ' +
-          computerChoice + ', The computer wins!')
+    playerChoice = input("Choose rock, paper or scisors\n>>> ")
+    if(playerChoice == 'end'):
+        print("Ending Game, We'll miss you!")
+        exit()
+    if(playerChoice == 'cheat'):
+        score += int(input("How much do you want to add to you score? "));
+        game()
+        
+    tc = playerChoice[0]
```

Comment:
oh yeah here it is

Reply:
me casually showing off my github skills 😎

Reply:
although they are completely useless in this context

Reply:
lol, That is pretty cool tho

Reply:
yeah, i want to actually contribute to a project in this club. I never used github other than for stupid things. Those repos are private ofc just in case I ever want a career in cs.